.github/workflows/generate_new_client.yaml

```diff
@@ -0,0 +1,178 @@
+        rest_docs:
+          required: false
+          type: string
+          description: |
+            `rest_docs`: If it exists, link to the REST Documentation for a service
+        rpc_docs:
+          required: false
+          type: string
+          description: |
+            `rpc_docs`: If it exists, link to the RPC Documentation for a service
+        api_description:
+          required: true
+          description: "`api_description`: Description that appears in README.md"
+        release_level:
+          required: false
+          type: choice
+          options:
+            - stable
+            - preview
+          description: |
+            `release_level`: A label that appears in repo-metadata.json.
+            The first library generation is always 'preview'.
+        transport:
+          required: false
+          type: choice
+          options:
+            - grpc
+            - http
+            - both
+          description: "`transport`: A label that appears in repo-metadata.json"
+        language:
+          required: false
+          type: string
+          default: java
+        distribution_name:
+          required: false
+          type: string
+          description: |
+            `distribution_name`: Maven coordinates of the generated library. By default it's
+            "com.google.cloud:google-cloud-<api_shortname>"
+        api_id:
+          required: false
+          type: string
+          description: |
+            `api_id`: The value of the apiid parameter used in README.md It has link to
+            "https://console.cloud.google.com/flows/enableapi?apiid=<api_id>"
+        requires_billling:
+          required: false
+          type: boolean
+          description: |
+            `requires_billling`: Based on this value, README.md explains whether billing setup is
+            needed or not.
+        destination_name:
+          required: false
+          type: string
+          description: |
+            `destination_name`: The directory name of the new library. By default it's
+            java-<api_shortname>
+        cloud_api:
+          required: false
+          type: boolean
+          description: |
+            `cloud_api`: If true, the artifact ID of the library is 'google-cloud-'
+            otherwise 'google-'
+        group_id:
+          required: false
+          type: string
+          description: |
+            `group_id`: The group ID of the artifact when distribution name is not set
+        owlbot_image:
+          required: false
+          type: string
+          description: "`owlbot_image`: The owlbot container image used in OwlBot.yaml"
+        library_type:
+          required: false
+          type: string
+          description: |
+            `library_type`: A label that appear in repo-metadata.json to tell how the library is
+            maintained or generated
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest

```

Comment:
Avoid using "latest" because it may change one day and stops working.

Reply:
using 22.04